Sprint-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,64 @@
-const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+let autoBackInterval;
+let autoforwardInterval;
+const imagesForSlider = [
+  "./assets/cute-cat-a.png",
+  "./assets/cute-cat-b.jpg",
+  "./assets/cute-cat-c.jpg",
 ];
 
+let seconds = 1;
 
-// Write your code here
\ No newline at end of file
+const prevButton = document.querySelector("#backward-btn"),
+  nextButton = document.querySelector("#forward-btn"),
+  autoForward = document.querySelector("#autoforward-btn"),
+  autoBack = document.querySelector("#auto"),
+  stopButton = document.querySelector("#stop-btn"),
+  image = document.querySelector("#carousel-img"),
+  secondsDelay = document.getElementById("seconds-input");
+
+let curIndex = 1;
+
+function showSlide(index) {
+  imagesForSlider.forEach((src, i) => {
+    if (i === index) {
+      image.src = imagesForSlider[i];
+    }
+  });
```

Comment:
Can you figure out a much simpler way to show the image at position `index`?

Reply:
good point, I just removed if check